news/tasks.py

```diff
@@ -787,13 +787,6 @@ def attempt_fix(ext_name, record, task, e):
 
 @et_task
 def send_recovery_message_task(email):
-    # Have to import here to avoid circular import - that means that for
-    # testing, this can't be mocked. Mock look_for_user instead.
-    from news.views import get_user_data
```

Comment:
Because the function was moved to `utils.py` and it seems this task was just never fixed. This was part of the upgrade to Django 1.8. It's imported at the top of the file and used by other tasks now.